src/ddmd/escape.d

```diff
@@ -169,7 +169,7 @@ bool checkParamArgumentEscape(Scope* sc, FuncDeclaration fdc, Identifier par, Ex
  * to eliminate the error.
  * Params:
  *      sc = used to determine current function and module
- *      ae = AssignExp to check for any pointers to the stack
+ *      ae = AssignExp or CatAssignExp to check for any pointers to the stack
```

Comment:
The name of the parameter is wrong, should be `e` instead of `ae`.

Reply:
`e` implies any Expression, while `ae` implies assignment expression, which it is.

Reply:
I'm fine with either name, I was pointing out that the function parameter in the code is named 'e' while in here it's ae, so DDOC will not match them.

Reply:
oh, ok